pygfx/helpers/_stats.py

```diff
@@ -0,0 +1,141 @@
+import time
+
+from .. import (
+    plane_geometry,
+    Mesh,
+    Group,
+    TextGeometry,
+    Text,
+    TextMaterial,
+    ScreenCoordsCamera,
+    MeshBasicMaterial,
+)
+
+
+class Stats(Group):
+    """A Stats helper which displays performance statistics such
+    as FPS and draw time on the screen.
+
+    Parameters
+    ----------
+    viewport : Viewport, Renderer
+        Required for positioning lines of text in the corner of
+        the screen.
+
+    """
+
+    def __init__(
+        self,
+        viewport,
+    ):
+        from .. import Viewport  # avoid circular import
+
+        super().__init__()
+
+        self._line_height = 16
+        font_size = 12
+        foreground = "#0f0"
+        background = "#020"
+        text_material = TextMaterial(color=foreground)
+
+        quad_geometry = plane_geometry()
+        quad_geometry.positions.data[..., :2] += [0.5, -0.5]
+
+        self.bg = Mesh(
+            quad_geometry,
+            MeshBasicMaterial(
+                color=background,
+                side="BOTH",
+                opacity=0.9,
+            ),
+        )
+        # refactor once text bounding boxes are available
+        self.bg.scale.set(90, self._line_height * 2.1, 1)
+        self.ms = Text(
+            TextGeometry(
+                text="",
+                screen_space=True,
+                font_size=font_size,
+                anchor="topleft",
+            ),
+            text_material,
+        )
+        self.fps = Text(
```

Comment:
I see this would benefit from the TextGeometry being able to do multi-line ;)

Reply:
In very big ways, yes

Reply:
* Also being able to add more measurements without managing `n` text objects.